packages/flutter/lib/src/foundation/binding.dart

```diff
@@ -117,6 +118,14 @@ abstract class BindingBase {
         name: 'exit',
         callback: _exitApplication,
       );
+      registerServiceExtension(
+          name: 'dumpCompilationTrace',
```

Comment:
uber-nit: match indentation with the previous call above.

Reply:
Done